.github/workflows/linting.yaml

```diff
@@ -0,0 +1,23 @@
+name: Linting
+
+# This action works with pull requests and pushes
+on: [push, pull_request]
+
+jobs:
+  prettier:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          # Make sure the actual branch is checked out when running on pull requests
+          ref: ${{ github.head_ref }}
+
+      # Prettier Action: https://github.com/marketplace/actions/prettier-action
+      # Prettier config: https://prettier.io/docs/en/configuration.html
+      - name: Prettify code
+        uses: creyD/prettier_action@v4.3
+        with:
+          # This part is also where you can pass other options, for example:
+          prettier_options: --write **/*.{js,json,md}
```

Comment:
--write for an action makes me a little uncomfortable, and of course it doesn't work on forked PRs (not a powerful enough GITHUB_TOKEN). So how does that work out?

Reply:
I think the failure is a result of that, https://github.com/ddev/remote-config/actions/runs/6243383462/job/16948758817?pr=14

Reply:
I've update the workflow.